src/components/templates/Camp/Camp.tsx

```diff
@@ -33,12 +32,11 @@ const Camp = () => {
 
   const campLocation = `${venue.name}`;
 
-  const location = useMemo(
-    () =>
-      isRoomModalOpen && selectedRoom ? selectedRoom?.title : campLocation,
-    [isRoomModalOpen, selectedRoom, campLocation]
-  );
-  useUpdateLocationEffect(user, location);
+  // const location = useMemo(
```

Comment:
This might need to come back, thought I could easily use the `useUpdateLocationEffect` only on the VenuePage, but rooms need a different behaviour